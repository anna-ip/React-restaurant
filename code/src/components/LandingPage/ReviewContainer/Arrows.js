import React from 'react'

const Arrows = ({ nextReview, prevRewiev }) => {
  return (
    <>
      <button onClick={prevRewiev}>
        <span>◀︎</span>
      </button>
      <button onClick={nextReview}>
        <span>▶︎</span>
      </button>
    </>
  )
}

export default Arrows
