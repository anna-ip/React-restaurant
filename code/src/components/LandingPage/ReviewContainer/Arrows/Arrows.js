import React from 'react'
import styles from './Arrows.module.scss'

const Arrows = ({ nextReview, prevRewiev }) => {
  return (
    <>
      <button onClick={prevRewiev} className={styles.prev}>
        <span>←</span>
      </button>
      <button onClick={nextReview} className={styles.next}>
        <span>→</span>
      </button>
    </>
  )
}

export default Arrows
