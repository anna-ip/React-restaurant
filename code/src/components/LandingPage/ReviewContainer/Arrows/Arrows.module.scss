@import '../../../../styles/_variables.scss';

.prev,
.next {
  cursor: pointer;
  display: block;
  position: absolute;
  top: 177px;
  background-color: inherit;
  border: none;
  font-size: 6.4vw;
  color: $red-color;

  @media (min-width: $layout-breakpoint-small) {
    font-size: 4vw;
    top: 198px;
  }
}
.prev {
  left: 2vw;
  @media (min-width: $layout-breakpoint-small) {
    left: 1vw;
  }
}
.next {
  right: 2vw;
  @media (min-width: $layout-breakpoint-small) {
    right: 1vw;
  }
}
