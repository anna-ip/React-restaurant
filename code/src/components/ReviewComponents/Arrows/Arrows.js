import React from 'react'
import styles from './Arrows.module.scss'
import Button from '../../../ui-components/Button'

const Arrows = ({ nextReview, prevReview }) => {
  return (
    <>
      <Button onClick={prevReview} className={styles.prev} value={'←'} />
      <Button onClick={nextReview} className={styles.next} value={'→'} />
    </>
  )
}

export default Arrows
