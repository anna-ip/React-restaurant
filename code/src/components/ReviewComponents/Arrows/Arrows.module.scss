@import '../../../styles/_variables.scss';

.prev,
.next {
  display: block;
  position: absolute;
  top: 177px;
  font-size: 6.4vw;

  @media (min-width: $layout-breakpoint-small) {
    font-size: 4vw;
    top: 198px;
  }
}

.prev {
  left: 2vw;
  @media (min-width: $layout-breakpoint-small) {
    left: 1vw;
  }
}

.next {
  right: 2vw;
  @media (min-width: $layout-breakpoint-small) {
    right: 1vw;
  }
}
