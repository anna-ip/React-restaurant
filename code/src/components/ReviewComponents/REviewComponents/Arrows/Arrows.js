import React from 'react'
import styles from './Arrows.module.scss'

const Arrows = ({ nextReview, prevReview }) => {
  return (
    <>
      <button onClick={prevReview} className={styles.prev}>
        <span>←</span>
      </button>
      <button onClick={nextReview} className={styles.next}>
        <span>→</span>
      </button>
    </>
  )
}

export default Arrows
