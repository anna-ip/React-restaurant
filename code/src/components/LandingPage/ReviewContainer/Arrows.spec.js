import React from 'react'
import renderer from 'react-test-renderer'
import Arrows from './Arrows'

test('Renders correctly', () => {
  const tree = renderer.create(<Arrows />).toJSON()
  expect(tree).toMatchSnapshot()
})
