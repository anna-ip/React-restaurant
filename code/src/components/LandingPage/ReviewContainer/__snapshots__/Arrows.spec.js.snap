// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
Array [
  <button
    className="prev"
  >
    <span>
      ←
    </span>
  </button>,
  <button
    className="next"
  >
    <span>
      →
    </span>
  </button>,
]
`;
